rules_daml/daml.bzl

```diff
@@ -25,26 +25,30 @@ def _daml_configure_impl(ctx):
     project_version = ctx.attr.project_version
     dependencies = ctx.attr.dependencies
     data_dependencies = ctx.attr.data_dependencies
+    module_prefixes = ctx.attr.module_prefixes
     daml_yaml = ctx.outputs.daml_yaml
     target = ctx.attr.target
     opts = ["--target={}".format(target)] if target else []
     ctx.actions.write(
         output = daml_yaml,
         content = """
-            sdk-version: {sdk}
-            name: {name}
-            version: {version}
-            source: .
-            data-dependencies: [{data_dependencies}]
-            dependencies: [{dependencies}]
-            build-options: [{opts}]
-        """.format(
+sdk-version: {sdk}
+name: {name}
+version: {version}
+source: .
+data-dependencies: [{data_dependencies}]
+dependencies: [{dependencies}]
+module-prefixes:
+{module_prefixes}
```

Comment:
the difference is that this one takes a dict/record, not a list. I could format the string as a oneliner json record, but went for yaml style to avoid escaping double quotes